spec/views/catalog/bulk_upload_form.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+RSpec.describe 'catalog/bulk_upload_form.html.erb', :type => :view do
+
+  before :each do
+    @obj = Dor.find 'druid:hv992ry2431'
```

Comment:
Could this be mocked using a double? eg.
```ruby
let(:current_object) { double('object', { id: 'druid:abc123' }) }
# then in the before statement you can assign it, removing the need to do so in each `it` block
before :each do
  assign(:object, current_object)
end
```
This would remove the need to call `Dor.find` and potentially separate the view spec from the Dor.find dependency (or that object being indexed).